src/main/java/productDependency/CreateCategoryPage.java

```diff
@@ -140,9 +140,10 @@ public void enterCategoryName() throws IOException {
 		CreateCategoryPage sp = new CreateCategoryPage(driver);
 		log.info("Enter Category Name");
 		Assert.assertTrue(enterCategoryName.isDisplayed());
-		enterCategoryName.sendKeys(BasePage.getCellData(xlsxName, sheetName, 3,0));
+		enterCategoryName.sendKeys(BasePage.getCellData(xlsxName, sheetName, 3, 0));
 	}
-
+	
+	@FindBy(how = How.XPATH, using = "//select[@name='entities']")
```

Comment:
@rinkykasyap Looks like a required XPATH. How come this was missing earlier?

Reply:
@vikasbo I am re-using this xpath, I have followed the standard architecture of our WDAutomation Project so I have removed this xpath from the line no. 155 and added it here.